.github/workflows/release.yml

```diff
@@ -43,8 +43,9 @@ jobs:
         git add dist/index.js
         git add dist/sourcemap-register.js
         git add src/template/index.ts
-        git commit -m "chore(): Updating dist"
-        git push
+        # This can fail if `dist` was already run in branch
+        git commit -m "chore(): Updating dist" | true
+        git push | true
```

Comment:
should these be `|| true`?  otherwise this is susceptible to `-o pipefail` changing the exit code.  perhaps even it should be `(git commit -m "..." && git push) || true`